__tests__/app.test.js

```diff
@@ -318,3 +318,25 @@ describe("GET:/api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles?sort_by=title&order=ASC", () => {
+  test("200: Responds with an array of article objects with correct properties ordered by given values", () => {
+    return request(app)
+      .get("/api/articles?sort_by=title&order=ASC")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBeGreaterThan(1);
+        expect(articles).toBeSortedBy("title", {
+          descending: false,
+        });
+      });
+  });
+  test("400: Responds with a status 400 if given an invalid sort_by or order", () => {
+    return request(app)
+      .get("/api/articles?sort_by=invalid_sort_by&order=invalid_order")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad request");
+      });
```

Comment:
We're playing around with 2 different queries here `sort_by` and `order`... what happens if just sort_by is invalid? Or just order? The error handling here could only be for one or the other. Consider breaking this down into each part separately. You could also consider this for the success responses if just `sort_by` or `order` is queried.